ush/interp_tools.py

```diff
@@ -25,7 +25,7 @@ def date_range(current_day, ebb_dcycle, persistence):
     
     if ebb_dcycle == 1:
        print('Find  RAVE for ebb_dcyc 1')
-       if persistence == True:
+       if persistence:
```

Comment:
@JohanaRomeroAlvarez - The `persistence` block is now entered. I attached a `smoke_dust` log [smoke_dust_2019072200.log](https://github.com/user-attachments/files/17603647/smoke_dust_2019072200.log) that shows this. It probably makes sense for you to test, since I'm not sure what behavior is expected. Mostly I'm wondering if persistence should apply on the first day and the dummy emissions file should be input: `First day true, no RAVE files available. Use dummy emissions file`.